include/lemlib/odom/odom.hpp

```diff
@@ -31,7 +31,7 @@ class Odom {
          */
         void setPose(Pose pose);
     protected:
-        // heading set to M_PI_2 because odom should default to IMU 0 angle
-        Pose pose = Pose(0, 0, M_PI_2);
+        // heading set to 90 degrees because odom should default to IMU 0 angle
+        Pose pose = Pose(0_in, 0_in, 90_deg);
```

Comment:
```suggestion
        // heading set to `pid/2` radins because odom should default to IMU 0 angle
        Pose pose = Pose(0_in, 0_in, M_PI_2_rad);
```
change to radians for consistency